features/logging/4.x/elasticsearch_operator.feature

```diff
@@ -0,0 +1,32 @@
+@clusterlogging
+Feature: elasticsearch-operator related tests
+
+  # @author qitang@redhat.com
+  # @case_id OCP-21332
+  @admin
+  @destructive
+  @commonlogging
+  Scenario: ServiceMonitor Object for Elasticsearch is deployed along with the Elasticsearch cluster
+    Given I switch to cluster admin pseudo user
+    Given I use the "openshift-monitoring" project
+    And I run the :serviceaccounts_get_token client command with:
+      |serviceaccount_name | prometheus-k8s |
+    Then the step should succeed
+    And evaluation of `@result[:response]` is stored in the :token clipboard
+
+    Given I use the "openshift-logging" project
+    When I get project servicemonitor named "monitor-elasticsearch-cluster" as YAML
+    Then the step should succeed
+    And the output should contain:
+      | port: elasticsearch        |
+      | path: /_prometheus/metrics |
```

Comment:
We have a class `ServiceMonitor` already.  We better have here:
```
Given the expression should be true> service_monitor("monitor-elasticsearch-cluster").port == "elasticsearch"
And the expression should be true> service_monitor("monitor-elasticsearch-cluster").path == "/_prometheus/metrics"
```
`path` and `port` methods need to be written. I'll merge for now though as this is not critical here.

Reply:
If using these non-exist methods, the case would fail. I think that's great to have `path` and `port` methods, I'll keep these steps in this scenario, and once we have the methods for `servicemonitor` object, I'll update the scenarios. What do you think?

Reply:
@akostadinov  I wrote the `port` and `path` methods, PTAL, thanks.